cmake/projects/spdlog/hunter.cmake

```diff
@@ -108,6 +108,17 @@ hunter_add_version(
     61136ee6120fe069d37df4ad11628a2a0622b447
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    spdlog
+    VERSION
+    "1.4.2-p2"
+    URL
+    "https://github.com/cpp-pm/spdlog/archive/v1.4.2-p2.tar.gz"
```

Comment:
This updating of `spdlog`'s `v1.x` branch is confusing. Let's use the convention of last known release version + git hash for the updated version in Hunter. I created the new branch and release for you here: https://github.com/cpp-pm/spdlog/releases/tag/v1.4.2-58e6890-p0 and deleted the other, confusing branches.

For new versions, the branch name should be `hunter-<version>` without the `p0`. The release+tag adds the `p0`, `p1`, `p2`, etc. so that you can continue patching the same branch but have multiple releases.

Reply:
you are right, that is much  better. Thank you for removing the wrong branches and creating a new one.

I'll update the PR as soon as I find the time for it.